contrib/win32/openssh/appveyor.psm1

```diff
@@ -439,7 +493,41 @@ function Deploy-OpenSSHTests
     Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Include *.ps1,*.psm1 -Force -ErrorAction Stop
 
     $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "bin\$folderName\$RealConfiguration"    
-    Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Exclude ssh-agent.exe, sshd.exe -Force -ErrorAction Stop    
+    Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Exclude ssh-agent.exe, sshd.exe -Force -ErrorAction Stop
+
+
+    $sshdConfigFile = "$OpenSSHTestDir\sshd_config"
+    if (-not (Test-Path -Path $sshdConfigFile -PathType Leaf))
+    {
+        Write-BuildMessage "Installation dependencies: $OpenSSHTestDir\sshd_config is missing in the folder" -Category Error
+        throw "$OpenSSHTestDir\sshd_config is missing in the folder"
+    }
+
+    if ($env:DebugMode)
+    {
+        $strToReplace = "#LogLevel INFO"
+        (Get-Content $sshdConfigFile).Replace($strToReplace,"LogLevel Debug3") | Set-Content $sshdConfigFile
+    }
+    if(-not ($env:psPath))
+    {
+        $psCorePath = GetLocalPSCorePath
+        Set-BuildVariable -Name psPath -Value $psCorePath
+    }    
+
+    $strToReplace = "Subsystem	sftp	C:/Program Files/OpenSSH/sftp-server.exe"
+    if($env:psPath)
+    {
+        $strNewsubSystem = @"
+Subsystem	sftp	$OpenSSHTestDir\sftp-server.exe
```

Comment:
The indentation at this section in the sshd_config file (both linux and windows) is tab. I followed the same.